VTUL/data-repo
# add form field so that users can edit their display name

**Allows users to edit their display name.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-942) (:star:)

# What does this Pull Request do? (:star:)
Adds a form field that allows users to edit their name in their user profile. Kinda crazy that this wasn't in there.

# What's the changes? (:star:)
Adds a form field that allows users to edit their name in their user profile. Kinda crazy that this wasn't in there.

# How should this be tested?
* Log in
* Visit the user edit page. It's like this:
https://data-repo.dld.lib.vt.edu/users/<your_pid>@vt-dot-edu/edit
* Check that the first form field is `display_name`
* Edit your name once or twice to make sure that it works.

# Additional Notes:
* branch: `edit_username`


# Interested parties
@shabububu 

(:star:) Required fields
